### PR TITLE
Adding arenite

### PR DESCRIPTION
**Pull request by lcavadas**

### CODE REVIEW

File: data.js [hunk 1]
```diff
@@ -269,7 +269,7 @@ module.exports = [
   {
     name: "Muject",
     github: "srijs/muject",
-    tags: ["dependency", "injection", "inversion", "control", "ioc", "di", "require"],
+    tags: ["dependency", "injection", "inversion", "control", "ioc", "di", "require", ],
```
Comment (audiolion) [inline data.js]: Please revert this change

File: data.js [hunk 2]
```diff
@@ -5057,5 +5057,13 @@ module.exports = [
     description: "Animation micro library. Vanilla JavaScript.",
     url: "https://github.com/branneman/TinyAnimate",
     source: "https://raw.githubusercontent.com/branneman/TinyAnimate/master/src/TinyAnimate.js"
+  },
+  {
+     name: "arenite",
+     github: "arenite/arenite",
+     tags: ["dependency", "inversion", "control", "injection", "ioc", "di", "require", "dependency injection", "provider", "IOC", "microkernel", "module", "framework", "service locator"],
+     description: "Inversion of Control / Dependency Injection Library",
+     url: "https://github.com/arenite/arenite",
```
Comment (audiolion) [inline data.js]: Please resolve conflicts

File: data.js (hunk 1 above)
Comment (lcavadas) [inline data.js]: Done

File: data.js (hunk 2 above)
Comment (lcavadas) [inline data.js]: Done
